modules/arm_plugin/armplg_build.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/sh
+
+BUILD_JOBS=${BUILD_JOBS:-$(nproc)}
+BUILD_TYPE=${BUILD_TYPE:-Release}
+UPDATE_SOURCES=${UPDATE_SOURCES:-clean}
+
+DEV_HOME=`pwd`
+OPENCV_HOME=$DEV_HOME/opencv
+OPENVINO_HOME=$DEV_HOME/openvino
+OPENVINO_CONTRIB=$DEV_HOME/openvino_contrib
+ARM_PLUGIN_HOME=$OPENVINO_CONTRIB/modules/arm_plugin
+STAGING_DIR=$DEV_HOME/armcpu_package
+
+
+
+fail()
+{
+    if [ $# -lt 2 ]; then
+      echo "Script internal error"
+      exit 31
+    fi
+    retval=$1
+    shift
+    echo $@
+    exit $retval
+}
+
+checkSrcTree()
+{
+    [ $# -lt 2 ] && fail
+
+    TEST_DIR_NAME=directory
+    if [ $# -gt 2 ]; then
+        TEST_DIR_NAME="$3 directory"
+    fi
+    if ! [ -d $1 ]; then
+        echo "Unable to detect $TEST_DIR_NAME at $1"
+        echo "Cloning master branch only..."
+        git lfs clone --recurse-submodules --single-branch --branch=master $2 $1 || fail 3 "Failed to clone $TEST_DIR_NAME. Stopping"
+    else
+        echo "Detected $TEST_DIR_NAME at $1"
+        echo "Considering it as source directory"
+        if [ "$UPDATE_SOURCES" = "reload" ]; then
+            echo "Source reloading requested"
+            echo "Removing existing sources..."
+            rm -rf $1 || fail 1 "Failed to remove. Stopping"
+            echo "Cloning master branch only..."
+            git lfs clone --recurse-submodules --single-branch --branch=master $2 $1 || fail 3 "Failed to clone $TEST_DIR_NAME. Stopping"
+        elif [ -d $1/build ]; then
+            echo "Build directory detected at $1"
+            if [ "$UPDATE_SOURCES" = "clean" ]; then
+                echo "Cleanup of previous build requested"
+                echo "Removing previous build results..."
+                rm -rf $2/build || fail 2 "Failed to cleanup. Stopping"
+            fi
+        fi
+    fi
+    return 0
+}
+
+
+
+#Prepare sources
+checkSrcTree $OPENCV_HOME https://github.com/opencv/opencv.git OpenCV
+checkSrcTree $OPENVINO_HOME https://github.com/openvinotoolkit/openvino.git OpenVINO
+checkSrcTree $OPENVINO_CONTRIB https://github.com/openvinotoolkit/openvino_contrib.git OpenVINO_contrib
+
+#cleanup package destination folder
+[ -e $STAGING_DIR ] && rm -rf $STAGING_DIR
+mkdir -p $STAGING_DIR
+
+#Build OpenCV
+mkdir -p $OPENCV_HOME/build && \
+cd $OPENCV_HOME/build && \
+PYTHONVER=`ls /usr/include | grep "python3[^m]*$"` && \
+cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DBUILD_LIST=imgcodecs,videoio,highgui,python3 \
+      -DBUILD_opencv_python2=OFF -DBUILD_opencv_python3=ON -DOPENCV_SKIP_PYTHON_LOADER=ON \
```

Comment:
Why do we add python dependency here?

Reply:
OpenVINO python samples require OpenCV as well. So it make sense either enable or disable python support for both. However IMO python is quite popular and worth adding to the generated package.